src/stylesheets/layouts/_concept.scss

```diff
@@ -0,0 +1,380 @@
+      align-self: center;
+      width: 220px;
+      top: 0;
+      transform: rotate(-180deg);
+      @media (min-width: 800px) {
+        width: 280px;
+        top: 71%;
+        transform: rotate(360deg);
+      }
+    }
+
+    &__img2 {
+      align-self: center;
+      width: 120px;
+      top: -13%;
+      @media (min-width: 550px) {}
+      @media (min-width: 800px) {
+        height: 180px;
+        width: 300px;
+        top: 74%;
+      }
+    }
+
+    &__img3-bg {
+      align-self: center;
+      width: 330px;
+      top: 0;
+      @media (min-width: 800px) {
+        width: 330px;
+        top: 0;
+        right: 40%;
+      }
+    }
+
+    &__img3 {
+      align-self: center;
+      height: 180px;
+      top: -15%;
+      transform: rotate(-33deg);
+      @media (min-width: 800px) {
+        height: 200px;
+        top: -11%;
+        right: 53%;
+      }
+    }
+
+    &__img4-bg {
+      align-self: center;
+      width: 330px;
+      top: 0;
+      @media (min-width: 800px) {
+        width: 500px;
+      }
+    }
+
+    &__img4 {
+      align-self: center;
+      width: 350px;
+      top: -15%;
+      transform: rotate(-10deg);
+      @media (min-width: 800px) {
+        width: 480px;
+        top: -7%;
+      }
+    }
+
+    &__img5-bg {
+      align-self: center;
+      width: 350px;
+      top: 0;
+      @media (min-width: 800px) {
+        width: 450px;
+      }
+    }
+
+    &__img5 {
+      align-self: center;
+      width: 155px;
+      top: -30%;
+      transform: rotate(11deg);
+      @media (min-width: 800px) {
+        width: 155px;
+        top: -11%;
+      }
+    }
+
+    &__img6-bg {
+      width: 0;
+      @media (min-width: 800px) {
+        width: 300px;
+        align-self: center;
+        bottom: 0;
+      }
+    }
+
+    &__img6 {
+      align-self: center;
+      width: 260px;
+      top: -26.5%;
+      clip: rect(0px,250px,120px,0px);
+      @media (min-width: 800px) {
+        clip: rect(0px, 1000px, 1000px, 0px);
+        width: 250px;
+        top: 68%;
+      }
+    }
+
+    &__img7-bg {
+      align-self: center;
+      width: 280px;
+      top: 0;
+      transform: rotate(-180deg);
+      @media (min-width: 800px) {
+        align-self: center;
+        width: 400px;
+        top: 58.2%;
+        transform: rotate(360deg);
+      }
+    }
+
+    &__img7 {
+      align-self: center;
+      width: 120px;
+      top: -30%;
+      transform: rotate(-18deg);
+      @media (min-width: 800px) {
+        align-self: center;
+        height: 220px;
+        top: 63%;
+        transform: rotate(10deg);
+      }
+    }
+  }
+}
+
+.bubble {
+  &__wrapper {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  &__florence {
+    display: inline-block;
+    border-radius: 1.5rem;
+    background-color: $color-bubble-mint;
+    margin: 0.2rem 18% 0.2rem 0.2rem;
+    padding: 0.8rem 1.2rem;
+    @media (min-width: 800px) {
+      margin: 0.2rem 10% 0.2rem 0.2rem;
+    }
+
+    &--tail {
```

Comment:
do you need this `--tail` element? Or can you put that `:before` on something else?